tensorflow_datasets/image/cartoonset_test.py

```diff
@@ -0,0 +1,40 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test for the cartoon dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import cartoonset
+
+class Cartoonset10kTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = cartoonset.Cartoonset
+  BUILDER_CONFIG_NAMES_TO_TEST = ["cartoonset10k"]
+  SPLITS = {
+      "train": 3,  # Fake training examples
+  }
+
+class Cartoonset100kTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = cartoonset.Cartoonset
+  BUILDER_CONFIG_NAMES_TO_TEST = ["cartoonset100k"]
+  SPLITS = {
+      "train": 30,  # Fake training examples
```

Comment:
30 test images is too much 3-4 is enough for testing.

Reply:
Yes - there are far too many test files in this PR.